### ISSUE TITLE
Unable to launch development server

### ISSUE THREAD

**Issue by ecognium** [Dalgona/Serum#164]
I am new to Serum and I was following the install instructions and I was able to build the default project but I am not able to launch the development server.  I get the following error:

```
could not start the Serum development server: {:failed_to_start_child, Microscope, {:EXIT, {:function_clause, 
[{Microscope.Options, :validate_field, [:extra_routes, [{"/serum_live_reloader", Serum.DevServer.LiveReloadHandler, nil}]], 
[file: 'lib/microscope/options.ex', line: 85]}, {Microscope.Options, :"-parse/1-fun-1-", 1, [file: 'lib/microscope/options.ex', 
line: 10]}, {Enum, :"-map/2-fun-0-", 3, [file: 'lib/enum.ex', line: 1403]}, {:maps, :fold_1, 3, [file: 'maps.erl', line: 233]},
 {Enum, :map, 2, [file: 'lib/enum.ex', line: 2197]}, {Microscope.Options, :parse, 1, [file: 'lib/microscope/options.ex', line: 10]}, 
{Microscope, :start_link, 2, [file: 'lib/microscope.ex', line: 95]}, {:supervisor, :do_start_child_i, 3, [file: 'supervisor.erl', 
line: 385]}]}}}
```

This is what I have done so far:

```
mix archive.install hex serum_new
mix serum.new test_site
cd test_site
mix do deps.get, deps.compile
mix serum.build ## this works
mix serum.server  ## this fails
```

```
~/dev/test_site|⇒  mix -v
Erlang/OTP 23 [erts-11.1.1] [source] [64-bit] [smp:12:12] [ds:12:12:10] [async-threads:1] [hipe] [dtrace]

Mix 1.11.1 (compiled with Erlang/OTP 23)
~/dev/test_site|⇒  elixir -v
Erlang/OTP 23 [erts-11.1.1] [source] [64-bit] [smp:12:12] [ds:12:12:10] [async-threads:1] [hipe] [dtrace]

Elixir 1.11.1 (compiled with Erlang/OTP 23)
```

**I am using macOS Catalina** 

I have other Phoenix projects and they all launch fine. I am not that proficient in Elixir to debug the quote/unquote blocks to see why it is failing. I did not make any changes to the generated project. 

Appreciate any help with the issue.  

Thanks!

**Comment by Dalgona** [comment]
Probably that error was caused by recent updates to one of Serum's dependencies.

As a temporary workaround, try adding the following dependency in your `mix.exs`:

```elixir
defp deps do
  [
    # ...
    {:microscope, "1.3.0", override: true}
    # ...
  ]
end
```

And then re-run `mix deps.get`.

**Comment by ecognium** [comment]
Thank you so much for the quick response. That indeed fixed the issue. If you want me to create a PR for the docs to include this info, happy to help.

**Comment by rocketbop** [comment]
@Dalgona Do you want any help with this

**Comment by Dalgona** [comment]
Just published Serum v1.5.1. Try updating Serum and remove the workaround (if you have done it).

```diff
# In your mix.exs:

defp deps do
   [
-    {:serum, "x.y.z"},
-    {:microscope, "1.3.0", override: true},
+    {:serum, "1.5.1"},
     # ...
   ]
end
```

**Comment by rocketbop** [comment]
Thanks @Dalgona ! Much appreciated...
